android/filament-android/src/main/java/com/google/android/filament/View.java

```diff
@@ -252,6 +264,28 @@ public void setDynamicLightingOptions(float zLightNear, float zLightFar) {
         nSetDynamicLightingOptions(getNativeObject(), zLightNear, zLightFar);
     }
 
+    public void setSSAO(@NonNull SSAO ssao) {
```

Comment:
How did we name equivalent methods in the Java bindings? `setSsao()` or `setSSAO()`? We should be consistent but I don't remember what we picked.

Reply:
I think if we wanted to be consistant it would have to be `setScreenSpaceAmbientOcclusion()` -- generally we don't abbreviate, but this was was a mouthful, so I broke the rule.

Reply:
It's fine to use abbreviations.

Reply:
I'm just saying we have not. e.g. we have `setAntiAliasing()`, not `setAA()`. Generally, filament's APIs don't abbreviate, so I don't think there is a precedent. I'm fine with `setScreenSpaceAmbientOcclusion()` btw, at least it's clear what it is.

Reply:
Actually it should be setAmbientOcclusion, with SSAO as an option. It could be non-screen space based someday.

Reply:
Okay, sounds good. done.